pcsx2/DEV9/ATA/Commands/ATA_SCE.cpp

```diff
@@ -24,6 +24,16 @@ void ATA::HDD_SCE()
 
 	switch (regFeature)
 	{
+		case 0xF1:
+		case 0xF2:
+		case 0xF3:
+		case 0xF4:
+		case 0xF5:
```

Comment:
Maybe add comments as to what these commands would be, or else add the feature value for the associated command in the comment block below.

Reply:
sure, I can move comments from lines 39-46 here

Reply:
I was more thinking of commenting on each case what the command was, as I had assumed you knew what command was triggered by what value.

Reply:
The rest of commands is not implemented and not documented, I added them for documentation purpose. But as they exist inside firmware they can be used by some game or app and it will be better to differentiate existing command from non-existent one.

Reply:
I was more asking for something like this
```c
		case 0xF1: // ATA_SCE_SECURITY_SET_PASSWORD
		case 0xF2: // ATA_SCE_SECURITY_UNLOCK
...
```
I note that some of the values differ between my original comment and what [PS2 Dev Wiki](https://www.psdevwiki.com/ps2/Hard_Drive) has.
I'm inclined to trust the values on PS2 Dev Wiki over what I had, I don't recall where I got the values I had noted before.